src/components/workspace/controlBar/chapterSelect.tsx

```diff
@@ -38,10 +39,35 @@ export function ChapterSelect(props: ChapterSelectProps) {
   );
   const ChapterSelectComponent = Select.ofType<IChapter>();
 
+  const chapterSelector = (currentChap: number, currentVariant: Variant, isClickable: boolean) => {
+    const stopPopup = (e: any) => {
+      e.stopPropagation();
+    };
+
+    return isClickable ? (
+      <div>
+        <Button
+          className={Classes.MINIMAL}
+          text={styliseChapter(currentChap, currentVariant)}
+          rightIcon={IconNames.DOUBLE_CARET_VERTICAL}
```

Comment:
We can inline the value of `rightIcon` with a ternary statement on the value of `isClickable`.

Reply:
Aights!